src/AddToMetaMask.tsx

```diff
@@ -33,6 +33,8 @@ const AddToMetaMask = (props: {
                 blockExplorerUrls: props.be,
               },
             ],
+          }).then(response => {
+            if (response === null) alert('Added')
```

Comment:
Can you change the alert to something like:
```
The RPC endpoint for Chain ID {props.chainId} already exists.
If you want to replace it, please remove the old RPC endpoint first and then add the new one.
```

```suggestion
            if (response === null) alert('Added')
```

Reply:
- This message is shown when you first add Sapphire too. We would need to measure how quickly null is returned to differentiate "already exists" vs "added first time".
- If you have OPF sapphire RPC added and you click 1RPC: metamask will ask you if you want to add 1RPC as a separate entry. No need to remove first. (if we really wanted to, I guess we could detect any existing entries with wallet_switchEthereumChain)